templates/bootstrap/pyproject.toml.j2

```diff
@@ -11,7 +11,11 @@ filename = "CHANGES.rst"
 directory = "CHANGES/"
 title_format = "{version} ({project_date})"
 template = "CHANGES/.TEMPLATE.rst"
+{% if issue_tracker=='redmine' %}
 issue_format = "`#{issue} <https://pulp.plan.io/issues/{issue}>`_"
+{% else %}
+issue_format = "`#{issue} <https://github.com/pulp/{{ plugin_name }}/issues/{issue}>`_"
```

Comment:
Should we add support for different orgs here?

Reply:
I think so

Reply:
I think this might require a separate PR. There are other templates where we have Github links with the Pulp org.